src/VideoJS.as

```diff
@@ -94,39 +94,43 @@ package{
                 }
             }
             finally{}
-            
-            
-            
+
+
+
             setTimeout(finish, 50);
 
         }
-        
+
         private function finish():void{
-            
+
             if(loaderInfo.parameters.mode != undefined){
                 _app.model.mode = loaderInfo.parameters.mode;
             }
-            
-            if(loaderInfo.parameters.eventProxyFunction != undefined){
+
+            // Hard coding these in for now until we can come up with a better solution for 5.0 to avoid XSS.
+            _app.model.jsEventProxyName = 'videojs.Flash.onEvent';
```

Comment:
We need to remove these settings from core.
https://github.com/videojs/video.js/blob/master/src/js/media/flash.js#L39